src/client/Client.h

```diff
@@ -1238,6 +1242,9 @@ class Client : public Dispatcher, public md_config_obs_t {
 
   bool _use_faked_inos;
 
+  // Cluster fsid
+  int64_t fscid;
```

Comment:
Using a bare int64_t in the outward facing libcephfs bits makes sense, but here in Client I think we can use the fs_cluster_id_t typedef

Reply:
Fair enough. I'll change it then.